content/examples/constants.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "constants"
+date: 2023-06-25T14:50:00-06:00
+draft: false
+---
+
+Constants store data that cannot be modified, just like inmutable variables.
+
+The key differece are that constants are defined ad deploytime and
+- Defined with the keyworkd `const` and named in SCREAMING_SNAKE_CASE
+- Are only declared in the global scope.
```

Comment:
Is this because of convention? I think we should specify that.

Reply:
Aparently [it is](https://github.com/cairo-book/cairo-book.github.io/blob/main/src/ch02-01-variables-and-mutability.md?plain=1#L118)! Wasn't really sure abou it.